pkg/webhooks/identity/identity.go

```diff
@@ -121,7 +121,7 @@ func (s *IdentityWebhook) authorized(request admissionctl.Request) admissionctl.
 			}
 		}
 		log.Info("Denying access", "request", request.AdmissionRequest)
-		ret = admissionctl.Denied("Prevented from accessing Red Hat managed identity provider. Please manage your identity provider through https://cloud.redhat.com/openshift")
+		ret = admissionctl.Denied("Prevented from modifying Red Hat's managed Identity. You may create/modify any Identity objects, except for 'OpenShift_SRE'.")
```

Comment:
```suggestion
		ret = admissionctl.Denied(fmt.Sprintf("Prevented from modifying Red Hat's managed Identity. You may create/modify any Identity objects, except for %s.", DefaultIdentityProvider))
```

Reply:
Suggestion to implement for my [previous comment](https://github.com/openshift/managed-cluster-validating-webhooks/pull/116/files#r511300954).